src/tools/rbd/ArgumentTypes.h

```diff
@@ -45,6 +45,7 @@ static const std::string JOURNAL_SPEC("journal-spec");
 static const std::string PATH_NAME("path-name");
 
 // optional arguments
+static const std::string ALL_NAME("all");
```

Comment:
Nit: just declare/define this new argument within Snap.cc since that's the only place it's currently used. The fields are are re-used between other actions.